pyturbo/systems/generic/generic_simple_view.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2022-2024, twiinIT
+# SPDX-License-Identifier: BSD-3-Clause
+
+from cosapp.systems import System
+
+from pyturbo.ports import KeypointsPort, View, ViewPort
+
+
+class GenericSimpleView(System):
+    """Class with visual representation of a generic simple view.
+
+    Inputs
+    ------
+    kp : KeypointsPort
+        Geometrical envelop.
+
+    Outputs
+    -------
+    occ_view : ViewPort
+        The visualisation.
+    """
+
+    def setup(self, shell_view=False):
+        self.add_inward("shell_view", shell_view)
+
+        # inputs/outputs
+        self.add_input(KeypointsPort, "kp")
+        self.add_output(ViewPort, "occ_view")
+
+        # inwards
+        self.add_inward("n", 1, unit="", desc="stage count")
+
+    def compute(self):
+        shell = {
+            "shape": self.kp.view(self.shell_view),
+            "face_color": "white",
+            "opacity": 0.9,
+        }
+
+        view = View({"shell": shell})
+
+        self.occ_view.set_value(view)
```

Comment:
why not using property with getter/setter?

Reply:
`occ_view` need to be a port instead of a complex object ?

Reply:
The goal is to separate the port (used in Cosapp to manage boundary conditions, connections, etc.)  from the complex object that can be handled independently within the system itself.  
This approach limits the number of functions within the port, keeping it simpler.  
However, is this really relevant? I'm starting to have doubts...

Reply:
Using ports also allow to use the coordinate transfert function